src/controllers/controllermanager.cpp

```diff
@@ -274,6 +274,7 @@ void ControllerManager::slotSetUpDevices() {
         if (!pMapping) {
             continue;
         }
+        pMapping->restoreSettings(mappingFile, m_pConfig, pController->getName());
```

Comment:
At this point it is puzzling what restoreSettings() does. 

From the view of the mapping, it loads the stored settings form the config file. 
How about "loadSettings()" or such.